.github/workflows/test.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       max-parallel: 5
       matrix:
-        python-version: ['2.7', '3.5', '3.6', '3.7', '3.8']
+        python-version: [3.6, 3.7, 3.8, 3.9]
```

Comment:
Don't remove quotes here. This is a good pattern to make sure that the next version of 3.10 won't be interpreted wrong in yml.